packages/components/src/FormatRelativeDateTime/FormatRelativeDateTime.test.tsx

```diff
@@ -4,142 +4,130 @@ import { cleanup } from "@testing-library/react";
+  const testDate = new Date();
   testDate.setDate(testDate.getDate() - 60);
-
-  const civilTestDate = getCivilTime(testDate);
-  const isoTestDate = getISOString(testDate);
-
-  expect(
-    renderer.create(<FormatRelativeDateTime date={civilTestDate} />).toJSON(),
-  ).toMatchInlineSnapshot('"Apr 26"');
-  expect(
-    renderer.create(<FormatRelativeDateTime date={isoTestDate} />).toJSON(),
-  ).toMatchInlineSnapshot('"Apr 26"');
-  expect(
-    renderer.create(<FormatRelativeDateTime date={testDate} />).toJSON(),
-  ).toMatchInlineSnapshot('"Apr 26"');
+  const dates = getMockDates(testDate);
+
+  Object.values(dates).forEach(value => {
+    expect(
+      renderer.create(<FormatRelativeDateTime date={value} />).toJSON(),
+    ).toEqual(
+      new Date("Apr 26").toLocaleDateString(undefined, {
+        month: "short",
+        day: "numeric",
+      }),
+    );
+  });
 });
 
 it("renders the month and date when yesterday's date 1 year previous (border case)", () => {
-  const testDate = new Date(Date.now());
+  const testDate = new Date();
   testDate.setFullYear(testDate.getFullYear() - 1);
   testDate.setDate(testDate.getDate() + 2); //The +2 vs. +1 is a fudge for leap year
-
-  const civilTestDate = getCivilTime(testDate);
-  const isoTestDate = getISOString(testDate);
-
-  expect(
-    renderer.create(<FormatRelativeDateTime date={civilTestDate} />).toJSON(),
-  ).toMatchInlineSnapshot('"Jun 27"');
-  expect(
-    renderer.create(<FormatRelativeDateTime date={isoTestDate} />).toJSON(),
-  ).toMatchInlineSnapshot('"Jun 27"');
-  expect(
-    renderer.create(<FormatRelativeDateTime date={testDate} />).toJSON(),
-  ).toMatchInlineSnapshot('"Jun 27"');
+  const dates = getMockDates(testDate);
+
+  Object.values(dates).forEach(value => {
+    expect(
+      renderer.create(<FormatRelativeDateTime date={value} />).toJSON(),
+    ).toEqual(
+      new Date("Jun 27").toLocaleDateString(undefined, {
+        month: "short",
+        day: "numeric",
+      }),
+    );
+  });
 });
 
 it("renders the month day, year when over a year ago", () => {
-  const testDate = new Date(Date.now());
+  const testDate = new Date();
   testDate.setFullYear(testDate.getFullYear() - 2);
-
-  const civilTestDate = getCivilTime(testDate);
-  const isoTestDate = getISOString(testDate);
-
-  expect(
-    renderer.create(<FormatRelativeDateTime date={civilTestDate} />).toJSON(),
-  ).toMatchInlineSnapshot('"Jun 25, 2018"');
-  expect(
-    renderer.create(<FormatRelativeDateTime date={isoTestDate} />).toJSON(),
-  ).toMatchInlineSnapshot('"Jun 25, 2018"');
-  expect(
-    renderer.create(<FormatRelativeDateTime date={testDate} />).toJSON(),
-  ).toMatchInlineSnapshot('"Jun 25, 2018"');
+  const dates = getMockDates(testDate);
+
+  Object.values(dates).forEach(value => {
+    expect(
+      renderer.create(<FormatRelativeDateTime date={value} />).toJSON(),
+    ).toEqual(
+      new Date("Jun 25, 2018").toLocaleDateString(undefined, {
```

Comment:
Should `undefined` here be pulled out into a variable like in your other tests?

Reply:
nope. that one i'm testing differently than here. That one I actually cared about it and match it up with something, this one, not really

Reply:
gotcha! Thanks!